code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/views.cljs

```diff
@@ -830,7 +859,9 @@
         ;; TODO when implementing creation flow from apps page: Always show button and use temp-id for storing
         ;; and retrieving deployment-set and deployment-set-edited
         [:<>
-         [AddButton events/edges-picker-modal-id]
+         [AddButton {:modal-id events/edges-picker-modal-id
+                     :enabled  @can-edit?
+                     :tooltip  (when-not @can-edit? (@tr [:dep-group-edit-not-allowed]))}]
```

Comment:
When a user has only a view access to the deployment set. He will get this message on all buttons even if not in transitional state.

Reply:
Right.. the check needs to be more specific.

Reply:
I pushed another commit, now the specific message is only shown when the deployment group is in a transition state.
It is also shown to users that only have view access though. It would probably be better to give them the more generic message in all case, but I am not sure if we can make the distinction (between not editable because user has only view access and not editable when user has edit access but dep group in transition state) reliably.